services/121-service/migration/1713363871246-remove-startedRegistration-state.ts

```diff
@@ -1,18 +1,31 @@
 import { MigrationInterface, QueryRunner } from 'typeorm';
 
+const registrationStatus = 'startedRegistration';
+
 export class RemoveStartedRegistrationState1713363871246
   implements MigrationInterface
 {
   name = 'RemoveStartedRegistrationState1713363871246';
 
   public async up(queryRunner: QueryRunner): Promise<void> {
     console.time('RemoveStartedRegistrationState1713363871246');
+    await queryRunner.commitTransaction();
```

Comment:
Are `commitTransaction` and `startTransaction` part of the fix?

Reply:
Since there are multiple queries, so better to do them in a transaction.